openlr/openlr_sample.hpp

```diff
@@ -26,15 +26,23 @@ enum class ItemEvaluation
 
 struct SampleItem
 {
-  struct MWMSegemnt
+  struct MWMSegment
   {
+    MWMSegment(FeatureID const & fid, uint32_t const segId, bool const isForward)
+      : m_fid(fid)
+      , m_segId(segId)
+      , m_isForward(isForward)
+    {
+    }
+
     // TODO(mgsergio): switch to osm id.
     FeatureID m_fid;
     uint32_t m_segId;
+    bool m_isForward;
```

Comment:
There is no need to do this for the specific field or a subset of fields.
MWMSegment is a value object. If you want it to be const mark the whole object as const.